tests/tensorflow/save_keras_model.py

```diff
@@ -0,0 +1,63 @@
+import tensorflow as tf
+import sys
+import pandas as pd
+import numpy as np
+import pickle
+
+import mlflow.tensorflow
+from mlflow.utils.file_utils import TempDir
+
+tf.random.set_seed(1337)
+
+inputs = tf.keras.layers.Input(shape=3, name="features", dtype=tf.float64)
+outputs = tf.keras.layers.Dense(2)(inputs)
+model = tf.keras.Model(inputs=inputs, outputs=[outputs])
+
+task_type = sys.argv[1]
+save_as_type = sys.argv[2]
```

Comment:
```suggestion
save_as_type = sys.argv[2]
save_path = sys.argv[3]
```

Reply:
save_path only available when task_type == "save_model"

Reply:
You can pass a dummy value.